CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v1.0.0
+This is the first official version for the main-net deployment
```

Comment:
Add a period at the end of the sentence.

Reply:
Maybe the dependent versions in go.mod need to be updated?

Reply:
Waiting chain to release its tag.